docs/grammar/Ballerina.g4

```diff
@@ -342,6 +343,14 @@ actionInvocationStatement
     |   variableReferenceList '=' actionInvocation ';'
     ;
 
+transactionStatement
+    :   'transaction' '{' statement* '}' rollbackClause
+    ;
+
+rollbackClause
+    :   'onrollback' '{' statement* '}'
```

Comment:
This is fixed with https://github.com/ballerinalang/ballerina/pull/2307/commits/f8bd04f2552bdc4fdc923a16d64ffd4cdf3a2b5e